app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt

```diff
@@ -42,6 +44,7 @@ class ContentPortalView : CoordinatorLayout, ContentAdapter.ContentPanelListener
 
     sealed class ContentType {
         object News : ContentType()
+        object Ticket : ContentType()
```

Comment:
Just wonder the idea behind the scene to call it `Ticket`? Would it be better to call it `Product` as a more general item?

Reply:
This version is for e-voucher and ticket.
Basically, they are links to the e-commerce site, which contain tickets, vouchers, and maybe more in the future. I call them tickets just for easy identify "The links under voucher & ticket"

Reply:
Changed to Shopping Link